CHANGELOG.md

```diff
@@ -171,7 +173,7 @@
 <a name="v0.18.19"></a>
 ## [v0.18.19] - 2021-01-27
 ### Bug Fixes
-- **install:** ensure prompt response are handled correctly
+- **install:** ensure prompt respones are handled correctly
```

Comment:
Something is up with this. Had it happen yesterday too that the build job modified this to be an incorrect spelling. Weird.

Reply:
The workflow that comes after the release to spellcheck the changelog will fix it. Just not sure why it happens from running `make release`.